config/checkstyle-resources-suppressions.xml

```diff
@@ -82,6 +82,10 @@
     files="[\\/]InputOuterType.*\.java"/>
   <suppress checks="OuterTypeFilename"
     files="rule522classnames[\\/]InputClassNames\.java"/>
+  <suppress checks="OuterTypeFilename"
+    files="rule462horizontalwhitespace[\\/]InputWhitespaceAroundBasic\.java"/>
+  <suppress checks="OuterTypeFilename"
+    files="rule462horizontalwhitespace[\\/]InputWhitespaceAroundEmptyTypesAndCycles\.java"/>
```

Comment:
not ok, see comment above.
We should have no suppressions.

Reply:
done